tests/python_client/testcases/test_index.py

```diff
@@ -1087,6 +1087,57 @@ def test_index_collection_with_after_load(self):
         search_res, _ = collection_w.search(vectors, default_search_field, default_search_params, default_limit)
         assert len(search_res[0]) == ct.default_limit
 
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_turn_off_index_mmap(self):
+        """
+        target: disabling and re-enabling mmap for index
+        method: disabling and re-enabling mmap for index
+        expected: search success
+        """
+        self._connect()
+        c_name = cf.gen_unique_str(prefix)
+        collection_w, _ = self.collection_wrap.init_collection(c_name, schema=default_schema)
+        collection_w.insert(cf.gen_default_list_data())
+        collection_w.create_index(ct.default_float_vec_field_name, default_index_params, index_name=ct.default_index_name)
+        collection_w.alter_index(ct.default_index_name, {'mmap.enabled': True})
+        collection_w.load()
+        collection_w.release()
+        collection_w.alter_index(ct.default_index_name, {'mmap.enabled': False})
+        collection_w.load()
+        vectors = [[random.random() for _ in range(default_dim)] for _ in range(default_nq)]
+        collection_w.search(vectors[:default_nq], default_search_field,
+                            default_search_params, default_limit,
+                            default_search_exp)
+        collection_w.release()
+        collection_w.alter_index(ct.default_index_name, {'mmap.enabled': True})
+        collection_w.load()
+        collection_w.search(vectors[:default_nq], default_search_field,
+                            default_search_params, default_limit,
+                            default_search_exp)
+
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_drop_mmap_index(self):
+        """
+        target: disabling and re-enabling mmap for index
+        method: disabling and re-enabling mmap for index
+        expected: search success
+        """
+        self._connect()
+        c_name = cf.gen_unique_str(prefix)
+        collection_w, _ = self.collection_wrap.init_collection(c_name, schema=default_schema)
+        collection_w.insert(cf.gen_default_list_data())
+        collection_w.create_index(ct.default_float_vec_field_name, default_index_params,
+                                  index_name=ct.default_index_name)
+        collection_w.alter_index(ct.default_index_name, {'mmap.enabled': True})
+        collection_w.drop_index(index_name=ct.default_index_name)
+        collection_w.create_index(ct.default_float_vec_field_name, default_index_params,
+                                  index_name=ct.default_index_name)
+        collection_w.load()
+        vectors = [[random.random() for _ in range(default_dim)] for _ in range(default_nq)]
+        collection_w.search(vectors[:default_nq], default_search_field,
+                            default_search_params, default_limit,
+                            default_search_exp)
```

Comment:
here lack of the result check for search.

Reply:
> here lack of the result check for search.

I'll add the checks.